core/store/migrate/migrations/0189_create_automation_upkeep_state.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
+
+CREATE TABLE evm_upkeep_states (
+  id SERIAL PRIMARY KEY,
+  work_id TEXT NOT NULL,
+  evm_chain_id NUMERIC(20) NOT NULL,
+  upkeep_id NUMERIC(78) NOT NULL, -- upkeep id is an evm word (uint256) which has a max size of precision 78
+  completion_state SMALLINT NOT NULL,
+  ineligibility_reason SMALLINT NOT NULL,
+  block_number BIGINT DEFAULT 0 NOT NULL,
```

Comment:
Do we expect it to ever default to zero? When would that happen?

Reply:
+1
The block number is int64